Stats page first iteration

almossawi opened the pull request:
Made a start on Stats page charts, the intention being to add the content using dummy data and then refine or correct any mistakes thereafter.

almossawi committed:
Stats page, added donut with dummy data and left-hand side menu

almossawi committed:
Fixed map and stats page choice menu quirk

almossawi committed:
Added icons to donut

almossawi committed:
Stats page updates

* Added chart 2 header and country picker drop down,
* Used same convention as sgarrity for class and id names
* Chart 1 copy corresponds to selected priority (for demo purposes -- I
suspect the way I have done it is incorrect

sgarrity commented:
Bringing our discussion from [this Gist](https://gist.github.com/sgarrity/9975588) back into this PR:

I wondered if the syntax for percentages changes in different locales. Does the % symbol go before the number in some locales, have a space between the number and the symbol in some locales, etc.? We should ask @pascalchevrel or @flodolo.

I don't think that necessarily needs to block this PR though.

almossawi committed:
Changed the way donut prose is set

We set them all on json load now and then swap in the right one
according to the active choice

almossawi committed:
Fixed stats-panel html l10n wrappers per sgarrity

almossawi committed:
Modified donut prose wrappers per sgarrity

almossawi committed:
% literal is in template wrapper now

almossawi committed:
Added solid bg color to country picker drop down

pascalchevrel commented:
Yes, the percent can be before the number but it's very unusual (Persian /ex), the more frequent case is the space between the number and the percent sign, I think that not having a space in  between is actually the less frequent case.
